libs/ui-lib/lib/ocm/services/ClusterDetailsService.ts

```diff
@@ -45,7 +45,10 @@ const ClusterDetailsService = {
       },
     };
 
-    if (params.cpuArchitecture === CpuArchitecture.ARM || params.platform?.type === 'oci') {
+    if (
+      !values.isCMNSupported &&
```

Comment:
Wouldn't this mean that, if the value `isCMNSupported` becomes true, for OCI clusters it wouldn't set UMN=true, which seems to be necessary?

Reply:
BE already has an internal validation to set UMN=true when it hits. The problem here was that we were forcing this value from the UI

Reply:
Why are we setting UMN from the UI anyway? Is there a reason why BE doesn't do it in these particular cases?

Reply:
This is a question for BE I think.
@y0n1  WDYT?

Reply:
I think this happened in the past because the BE had to initialize this value to something. IMHO we should delegate the logic to the BE and not the other way around.